plugins/source/aws/client/testing.go

```diff
@@ -56,7 +58,15 @@ func AwsMockTestHelper(t *testing.T, parentTable *schema.Table, builder func(*te
 		t.Fatal(err)
 	}
 
-	plugin.ValidateNoEmptyColumns(t, tables, messages)
+	for _, table := range tables.FlattenTables() {
+		cols, _ := testOpts.SkipEmptyCheckColumns[table.Name]
+		if len(cols) > 0 {
+			continue // TODO ignore these columns but check the rest
```

Comment:
This needs a later fix... Maybe something like
`plugin.ValidateNoEmptyColumns(t, schema.Tables{table}, messages, cols...)` (`except` columns)